src/components/Settings/Customization/CustomizationAccordeon.tsx

```diff
@@ -0,0 +1,114 @@
+import { Accordion, Grid, Group, Stack, Text } from '@mantine/core';
+import { IconBrush, IconChartCandle, IconDragDrop, IconLayout } from '@tabler/icons';
+import { useTranslation } from 'next-i18next';
+import { ReactNode } from 'react';
+import { GridstackConfiguration } from './Layout/GridstackConfiguration';
+import { LayoutSelector } from './Layout/LayoutSelector';
+import { BackgroundChanger } from './Meta/BackgroundChanger';
+import { FaviconChanger } from './Meta/FaviconChanger';
+import { LogoImageChanger } from './Meta/LogoImageChanger';
+import { BrowserTabTitle } from './Meta/MetaTitleChanger';
+import { DashboardTitleChanger } from './Meta/PageTitleChanger';
+import { ColorSelector } from './Theme/ColorSelector';
+import { CustomCssChanger } from './Theme/CustomCssChanger';
+import { DashboardTilesOpacitySelector } from './Theme/OpacitySelector';
+import { ShadeSelector } from './Theme/ShadeSelector';
+
+export const CustomizationSettingsAccordeon = () => {
+  const items = getItems().map((item) => (
+    <Accordion.Item value={item.id} key={item.label}>
+      <Accordion.Control>
+        <AccordionLabel {...item} />
+      </Accordion.Control>
+      <Accordion.Panel>
+        <Text size="sm">{item.content}</Text>
+      </Accordion.Panel>
+    </Accordion.Item>
+  ));
+  return (
+    <Accordion variant="contained" chevronPosition="right">
+      {items}
+    </Accordion>
+  );
+};
+
+interface AccordionLabelProps {
+  label: string;
+  image: ReactNode;
+  description: string;
+}
+
+const AccordionLabel = ({ label, image, description }: AccordionLabelProps) => (
+  <Group noWrap>
+    {image}
+    <div>
+      <Text>{label}</Text>
+      <Text size="sm" color="dimmed" weight={400}>
+        {description}
+      </Text>
+    </div>
+  </Group>
+);
+
+const getItems = () => {
+  const { t } = useTranslation('settings/customization/general');
```

Comment:
Does that work? Can you use this translation in getItems?

Reply:
Yep, seems to work:
![image](https://user-images.githubusercontent.com/30572287/216825875-e5708d76-7d3e-4212-b65e-515f10e63b6d.png)